### PR TITLE
Add a new file "Play Some Cricket"

### PR DESCRIPTION
**Pull request by ukumar37**
Please review and merge my changes into master

### CODE REVIEW

**Comment by ukumar37** [comment]
Please review this, @casey-collab

File: english/play-some-cricket/playsomecricket.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Checkout most rare and funny moments incricket history.](https://youtu.be/fzqh6_90akI)
```
Comment (casey-collab) [inline english/play-some-cricket/playsomecricket.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/play-some-cricket/playsomecricket.md (hunk 1 above)
Comment (ukumar37) [inline english/play-some-cricket/playsomecricket.md]: I just added the link to the new file and commit-ed it.

Please review this, @casey-collab

**Comment by ukumar37** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Pull request merged**